sources/myapp.js

```diff
@@ -8,13 +8,19 @@ export default class MyApp extends JetApp {
 			version: VERSION,
 			router: BUILD_AS_MODULE ? EmptyRouter : HashRouter,
 			debug: !PRODUCTION,
-			start: "/top/start"
+			start: "/top/contacts"
 		};
 
 		super({...defaults, ...config});
 	}
 }
 
 if (!BUILD_AS_MODULE) {
-	webix.ready(() => new MyApp().render());
+	webix.ready(() => {
+		let app = new MyApp();
+		app.render();
+		app.attachEvent("app:error:resolve", () => {
+			webix.delay(() => app.show("/top/contacts"));
```

Comment:
may be add any message about why user redirected to this page?

Reply:
Added webix.message